src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -131,6 +135,39 @@ export const validationSchema_v2 = Yup.object()
   })
   .required();
 
+export const newVenueSchema = Yup.object()
+  .shape<NewVenueSchema>({
+    name: Yup.string()
+      .required("Name is required!")
+      .min(
+        VENUE_NAME_MIN_CHAR_COUNT,
+        ({ min }) => `Name must be at least ${min} characters`
+      )
+      .max(
+        VENUE_NAME_MAX_CHAR_COUNT,
+        ({ max }) => `Name must be less than ${max} characters`
+      )
```

Comment:
I believe we have extracted this out of other validation schemas into a reusable 'schema chunk', which should also be reused here.

Reply:
And in general, we should be reusing validations as much as possible (whether in their entirety, or smaller chunks of them), rather than duplicating them

Reply:
Noted.